introduction.md

```diff
@@ -1,3 +1,6 @@
 Ruby is my favorite Code School path
 
 This is a clone of a fork of the dojo_rules repo
+These are my deadly_skills
+* Git
+* JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?